### PR TITLE
Add Metrics to `static_stop_times` table

### PR DESCRIPTION
**Pull request by rymarczy**
This PR adds travel time and headway metrics to the `static_stop_times` table. 

These metrics allow for the retrieval of pre-computed schedule metrics information to be compared to similar metrics produced from GTFS-RT records. 

Three metrics have been added to the `static_stop_times` table. These metrics will be computed on each new GTFS static schedule that is processed.
- schedule_travel_time_seconds
- schedule_headway_trunk_seconds
- schedule_headway_branch_seconds

All metrics are calculated as `UPDATE FROM` PostgreSQL commands, so no data transfer is required to or from our ECS instance. 

Asana Task: https://app.asana.com/0/1204389777951216/1204449669814304

### CODE REVIEW

File: python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py [hunk 1]
```diff
@@ -313,7 +315,12 @@ def insert_data_tables(
     insert static gtfs data tables into rds tables
     """
     for table in static_tables.values():
+        process_logger = ProcessLogger(
+            "gtfs_insert", table_name=table.table_name
+        )
+        process_logger.log_start()
         db_manager.insert_dataframe(table.data_table, table.insert_table)
+        process_logger.log_complete()
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py:323]: Some extra logging to track how long each GTFS table is taking to load.

File: python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py [hunk 2]
```diff
@@ -347,6 +354,12 @@ def process_static_tables(db_manager: DatabaseManager) -> None:
             drop_bus_records(static_tables)
             insert_data_tables(static_tables, db_manager)
 
+            static_timestamp = int(
+                static_tables["feed_info"].data_table.loc[0, "timestamp"]
+            )
+
+            modify_static_tables(static_timestamp, db_manager)
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py:361]: Pull out the `timestamp` value from the Schedule being processed and pass to the `modify_static_tables` function for more processing.

File: python_src/src/lamp_py/performance_manager/l0_gtfs_static_mod.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+                    order_by=StaticStopTimes.stop_sequence,
+                )
+            ).label("schedule_travel_time_seconds"),
+        )
+        .select_from(StaticStopTimes)
+        .where(
+            StaticStopTimes.timestamp == fk_timestamp,
+        )
+        .cte("scheduled_travel_times")
+    )
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/l0_gtfs_static_mod.py:35]: These calls are all very similar, using window functions to calculate metrics and the performing and `UPDATE FROM` call on the RDS to update the columns.

File: python_src/src/lamp_py/performance_manager/l0_gtfs_static_mod.py [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+    """
+    travel_times_cte = (
+        sa.select(
+            StaticStopTimes.pk_id,
+            (
+                StaticStopTimes.arrival_time
+                - sa.func.lag(StaticStopTimes.departure_time).over(
+                    partition_by=StaticStopTimes.trip_id,
+                    order_by=StaticStopTimes.stop_sequence,
+                )
```
[the remark this answers is not in the document]
Comment (mzappitello) [reply python_src/src/lamp_py/performance_manager/l0_gtfs_static_mod.py:27]: ah. nevermind. `lag` goes in the other direction.

**Review by mzappitello — APPROVED** [review]

**Pull request merged**